group17-s2/a2/explanations.txt

```diff
@@ -0,0 +1 @@
+Apart from the bridge function all functionality was already implemented in a1. To implement said brige functionality, setfacl is used to grant one specified user read and execute permissions on the specified project. As already reasoned in a1 this cannot be easily achieved using standard linux permissions since we can only distinguish three enitities: the owner, a group an all others. Therefore one would have to add all users (including the brige users) to one group which has access to the project. Every time a new bridge user wants access to the project said group has to be changed which is not praticable.
```

Comment:
```suggestion
Apart from the bridge function all functionality was already implemented in a1. To implement said brige functionality, setfacl is used to grant one specified user read and execute permissions on the specified project. As already reasoned in a1 this cannot be easily achieved using standard unix permissions since we can only distinguish three enitities: the owner, a group an all others. Therefore one would have to add all users (including the brige users) to one group which has access to the project. Every time a new bridge user wants access to the project said group has to be changed which is not practicable.
```

Reply:
Letzten beiden Sätze nicht so geil